content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -37,6 +37,26 @@ expression** that will be executed in an environment that has access to the foll
 - `uid (int)`: the id of the current user;
 - `today (str)`: the current local date in the `YYYY-MM-DD` format;
 - `now (str)`: the current local datetime in the `YYYY-MM-DD hh:mm:ss` format.
+- `companies (dict)`: the information of the companies. The dict contains:
+  - `multi_company (bool)`: A boolean indicating whether the user has access to multiple companies.
+  - `allowed_ids`: The list of company IDs the user is allowed to connect to.
+  - `active_ids`: The list of company IDs the user is connected to (selected in the company
+    switcher dropdown).
+  - `active_id`: The ID of the main company selected (the one highlighted in the company switcher
+    dropdown and displayed in the navbar of the webclient).
+  - `has(id|ids, 'property', value)`: returns a boolean indicating whether there's a company with id
+    in ids for which field matches the given value. Note that the properties of the companies are
+    those sent by the server in the session info. If a new property is needed, the company function
+    _get_session_info can be overridden. For example, the following code will add the
+    company_code property:
```

Comment:
Forgot to hit the "start a review" button :sweat_smile: 

RST lists must be preceded and followed by a blank line, otherwise the render is broken.